React/Views/RCTWebView.m

```diff
@@ -248,6 +270,13 @@ - (void)webView:(__unused UIWebView *)webView didFailLoadWithError:(NSError *)er
 
 - (void)webViewDidFinishLoad:(UIWebView *)webView
 {
+  if (_messagingEnabled) {
+    NSString *source = [NSString stringWithFormat:
+      @"window.postMessage = function(message) {"
+      "  window.location = '%@://%@?' + encodeURIComponent(JSON.stringify(message));"
+      "};", RCTJSNavigationScheme, RCTJSPostMessageHost];
```

Comment:
Since this executes after the webview's content has loaded, I think it'd be good not to stomp on any definition of `postMessage` that the content may have made. In other words, preface this with `if (window.postMessage === undefined)`.